src/transformers/models/seggpt/modeling_seggpt.py

```diff
@@ -0,0 +1,1012 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch SegGpt model."""
+
+
+import collections.abc
+import math
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.utils.checkpoint
+from torch import nn
+from torch.nn import functional as F
+
+from ...activations import ACT2FN
+from ...modeling_outputs import (
+    BaseModelOutput,
+)
+from ...modeling_utils import PreTrainedModel
+from ...utils import (
+    ModelOutput,
+    add_start_docstrings,
+    add_start_docstrings_to_model_forward,
+    logging,
+    replace_return_docstrings,
+)
+from .configuration_seggpt import SegGptConfig
+
+
+logger = logging.get_logger(__name__)
+
+# General docstring
+_CONFIG_FOR_DOC = "SegGptConfig"
+
+# Base docstring
+_CHECKPOINT_FOR_DOC = "EduardoPacheco/seggpt-vit-large"
+_EXPECTED_OUTPUT_SHAPE = [3, 896, 448]
+
+
+SEGGPT_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "EduardoPacheco/seggpt-vit-large",
+    # See all SegGpt models at https://huggingface.co/models?filter=seggpt
+]
+
+
+def patchify(tensor: torch.Tensor, patch_size: int) -> torch.Tensor:
```

Comment:
pachify and unpatchify below model output definitions

Reply:
this wasn't resolved

Reply:
So sorry! I completely overlooked this comment. Fixed here https://github.com/huggingface/transformers/pull/27735/commits/43f2d34800cd0bb5a84804f8f9b3da2b0c2ef6fa